src/DBgetAllData.php

```diff
@@ -1,5 +1,5 @@
 <?php
-function DBgetAllData ($db)
+function DBgetAllDataUSA ($db)
```

Comment:
It is convention for functions to start with lowercase too - `class`es in Object Oriented programming start with capital letters so the different casing is useful to disambiguate these things & will become important when we start using Object-Oriented PHP later in the course.

`DB` in function names inconsistent with your add function, which also uses the DB. I would say having `DB` in the function names is redundant - kind of obvious these thing suse the DB, but also that's an implementation detail - it's named *how* the function works, when good function names are named for *what the function does*

I also don't like the extra spaces between your function names and the opening parenthesis of the parameters - the parentheses touching the name of the function is also a way to better sight-read the code - if it has a touching paren then "you can just see" that it's a function. 

```suggestion
function dbGetAllDataUSA($db)
```

Reply:
that makes sense, thanks!